LICENSE

```diff
@@ -186,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright {yyyy} {name of copyright owner}
+   Copyright 2016 trivago GmbH
```

Comment:
@suyograo is reviewing this plugin. @IngaFeick do you mind changing this to "Copyright (c) 2012–2016 Elasticsearch <http://www.elastic.co>"?